migration-tools/fix-md-heading-links.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```

Comment:
Do we need to commit these python scripts to source? I have a feeling we don't need this whole directory, correct?